config/samples/dataplane_v1beta1_openstackdataplanenodeset_bgp.yaml

```diff
@@ -25,15 +25,19 @@ spec:
         hostName: edpm-compute-0
         ansible:
           ansibleHost: 192.168.122.100
-          ansibleVars:
-            ctlplane_ip: 192.168.122.100
-            internalapi_ip: 172.17.0.100
-            storage_ip: 172.18.0.100
-            tenant_ip: 172.19.0.100
-            fqdn_internalapi: edpm-compute-0.example.com
   networkAttachments:
     - ctlplane
   nodeTemplate:
+    networks:
+      - name: CtlPlane
+        subnetName: subnet1
+        defaultRoute: true
```

Comment:
You need to add the `fixedIP` here and other files changd here (same what's used for `ansibleHost`) as this is pre-provisioned. Else we would have issues with reserved ip vs `ansibleHost` var. Also, we should make the same change in https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml (which is bug I mentioned earlier) and drop https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml

Reply:
Ahh obviously, thanks for catching that.

ack on the second point. That sounds good.

Reply:
Sorry, I missed this earlier. `networks` in `nodeTemplate` is meant for situations where you don't want predictable IP at all (even for ctlplane). So using fixedIP here is incorrect. The correct thing would be to add it on the node section.